### DISCUSSION TITLE
how do I enable source maps for core files bundled with webapck

### DISCUSSION THREAD

**Discussion by ioweb-gr** [PrestaShop/PrestaShop#35615] [Q&A] [answered]
I'm working on a production installation of Prestashop 1.7.8.9  and I need to debug an issue from core.js file. Unfortunately this is bundled and minified without any source maps and it's impossible to troubleshoot. 

I cloned my environment, and disabled smart cache for javascript and css however, still no source maps are generated.

How do I build the file with source maps enabled to actually debug the issue?

**Comment by kpodemski** [comment, marked as answer]
Use the source files :)

https://github.com/PrestaShop/PrestaShop/tree/1.7.8.x/themes
